tests/test_constitutive_laws.py

```diff
@@ -0,0 +1,92 @@
+"""Tests for the _concrete_material_properties module"""
+import math
+import numpy as np
+
+import pytest
+
+from structuralcodes.materials.constitutive_laws import Elastic
+from structuralcodes.materials.constitutive_laws import ElasticPlastic
+from structuralcodes.materials.constitutive_laws import ParabolaRectangle
+from structuralcodes.materials.constitutive_laws import UserDefined
+
+
+@pytest.mark.parametrize(
+    'E, strain, expected',
+    [(210000, 0.003, 210000 * 0.003), (200000, 0.002, 200000 * 0.002)],
```

Comment:
Consider to add test of negative value.

Reply:
Actually, @talledodiego, I was thinking of testing the response to negative strains 😃

Reply:
done.